neqo-common/src/hrtime.rs

```diff
@@ -119,14 +119,13 @@ mod mac {
     #[repr(C)]
     #[derive(Debug, Copy, Clone, Default)]
     pub struct thread_time_constraint_policy {
-        period: f64,
-        computation: f64,
-        constraint: f64,
+        period: u32,
+        computation: u32,
+        constraint: u32,
```

Comment:
It will be this bit.  Sorry I missed it.  This is passed to a C interface, so we can't change the types.